albums.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel= "stylesheet" href= "/main.css">
+    <title>Document</title>
```

Comment:
Can you change the title to "Albums"? thx

Reply:
![fullsizerender-1](https://user-images.githubusercontent.com/30377594/37438405-3058cf1c-27c0-11e8-8305-5a0ef514bcfe.jpg)

Reply:
👍 +1